requirements.txt

```diff
@@ -0,0 +1,44 @@
+certifi==2023.5.7
+cffi==1.15.1
+charset-normalizer==3.2.0
+ci-info==0.3.0
+click==8.1.4
+configobj==5.0.8
+configparser==6.0.0
+etelemetry==0.3.0
+filelock==3.12.2
+fitz==0.0.1.dev2
+future==0.18.3
+httplib2==0.22.0
+idna==3.4
+importlib-resources==6.0.0
+isodate==0.6.1
+looseversion==1.3.0
+lxml==4.9.3
+networkx==3.1
+nibabel==5.1.0
+nipype==1.8.6
+numpy==1.24.4
+packaging==23.1
+pandas==2.0.3
+pathlib==1.0.1
+Pillow==10.0.0
+prov==2.0.0
+pycparser==2.21
+pydot==1.4.2
+pyheif==0.7.1
+PyMuPDF==1.22.5
+pyparsing==3.1.0
+python-dateutil==2.8.2
+pytz==2023.3
+pyxnat==1.5
+rdflib==6.3.2
+requests==2.31.0
+scipy==1.10.1
+simplejson==3.19.1
+six==1.16.0
+traits==6.3.2
+tzdata==2023.3
+urllib3==2.0.3
+zipp==3.16.0
+frontend==0.0.3
```

Comment:
Are you sure that they are all needed? Please put only those are needed for this library.